server/risksAndNeeds/risksAndNeedsController.test.ts

```diff
@@ -360,13 +360,13 @@ describe('Alcohol Misuse', () => {
 
 describe('Drug details section of risks and needs', () => {
```

Comment:
```suggestion
describe('Drug misuse section of risks and needs', () => {
```
sorry for nitpick but just to keep consistent with section name